beginner_source/deploy_seq2seq_hybrid_frontend_tutorial.py

```diff
@@ -52,23 +52,20 @@
 # operations called along the execution route taken by the example input
 # will be recorded. In other words, the control flow itself is not
 # captured. To convert modules and functions containing data-dependent
-# control flow, a **scripting** mechanism is provided. Scripting
-# explicitly converts the module or function code to Torch Script,
-# including all possible control flow routes. To use script mode, be sure
-# to inherit from the the ``torch.jit.ScriptModule`` base class (instead
-# of ``torch.nn.Module``) and add a ``torch.jit.script`` decorator to your
-# Python function or a ``torch.jit.script_method`` decorator to your
-# module’s methods. The one caveat with using scripting is that it only
-# supports a restricted subset of Python. For all details relating to the
-# supported features, see the Torch Script `language
-# reference <https://pytorch.org/docs/master/jit.html>`__. To provide the
-# maximum flexibility, the modes of Torch Script can be composed to
-# represent your whole program, and these techniques can be applied
-# incrementally.
-#
-# .. figure:: /_static/img/chatbot/pytorch_workflow.png
-#    :align: center
-#    :alt: workflow
+# control flow, a **scripting** mechanism is provided. The
+# ``torch.jit.script`` function takes module or function and does not
+# requires example inputs. Scripting then explicitly converts the module
+# or function code to TorchScript, including all possible control flow
+# routes. The one caveat with using scripting is that it only supports
```

Comment:
probably "all control flow" instead of "all possible control flow routes"?

Reply:
Also, I think "One caveat with using scripting" is a bit more natural than "The one caveat..."